src/coreclr/vm/nativelibrary.cpp

```diff
@@ -107,7 +107,7 @@ namespace
 #if defined(__APPLE__)
             COMPlusThrow(kDllNotFoundException, IDS_EE_NDIRECT_LOADLIB_MAC, libraryNameOrPath.GetUnicode(), GetMessage());
 #elif defined(TARGET_UNIX)
-            COMPlusThrow(kDllNotFoundException, IDS_EE_NDIRECT_LOADLIB_LINUX, libraryNameOrPath.GetUnicode(), GetMessage());
+            COMPlusThrow(kDllNotFoundException, IDS_EE_NDIRECT_LOADLIB_LINUX, libraryNameOrPath.GetUnicode());
```

Comment:
`IDS_EE_NDIRECT_LOADLIB_LINUX` error message expects two arguments, and this is going to give it only one argument now. I am not sure what the formatting is going to look like, but it is unlikely to be something good.